packages/x-components/src/components/result/base-result-rating.vue

```diff
@@ -76,7 +76,6 @@
 
 <style scoped lang="scss">
   .x-result-rating {
```

Comment:
Not related with this PR, but we have to be careful with the new result rating. In the demo view it is placed as a child another `a` element, which is [invalid HTML](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element).